### DISCUSSION TITLE
更新v1.8.7 后  不可U站点  每天都会因为频繁访问  变成429

### DISCUSSION THREAD

**Discussion by torashen** [jxxghp/MoviePilot#2084] [问题咨询]
更新v1.8.7 后  不可U站点  每天都会因为频繁访问  变成429
能请教下是什么原因么
站点里面已经设置了流控了
会访问站点的几个插件，自动签到、站点数据统计，以及刷流任务  其中刷流任务访问频率无法控制，其他2个设置签到为默认，站点数据统计为每6小时。。。 请教下这个是还会是其他什么原因么？

**Comment by kiri-to** [comment]
是不是做种列表里有种子名包含' > '
![image](https://github.com/user-attachments/assets/ac240d92-646e-4a17-8317-4cef81ab3c4c)
